packages/react-components/react-progress/Spec.md

```diff
@@ -2,62 +2,104 @@
+| Ant Design              | Progress           | [Progress](https://ant.design/components/progress/)                                                          | Specifies the type, a combined progress component                                                 |
+| Atlassian Design        | Progress bar       | [ProgressBar](https://atlassian.design/components/progress-bar/success-progress-bar/examples)                | Defaults to determinate state, which turns green when value reaches 1. Has an indeterminate state |
+| Bootstrap               | Progress           | [Progress](https://getbootstrap.com/docs/4.3/components/progress/)                                           | Allows for multiple different animation styles on the same Progress bar                           |
+| Carbon Design           | Progress Indicator | [ProgressIndicator](https://react.carbondesignsystem.com/?path=/story/components-progressindicator--default) | Determinate Progress that has steps, similar to a Slider                                          |
+| Fast                    | Progress           | [Progress](https://explore.fast.design/components/fast-progress)                                             | Combined Progress and Spinner component, has a determinate and indeterminate state                |
+| Lightning Design System | Progress Bar       | [ProgressBar](https://www.lightningdesignsystem.com/components/progress-bar/)                                | Has a vertical bar, only determinate, and can specify progress step                               |
+| Semantic UI             | Progress           | [Progress](https://semantic-ui.com/modules/progress.html#indicating)                                         | Allows for success and error states, default rounded edges. No indeterminate form                 |
+
+### Comparison of v8 and v0
+
+The existing components are:
+
+- v8
+  - `ProgressIndicator`
+- v0
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+Basic example:
+
+```jsx
+import { Progress } from '@fluentui/react-progress';
+
+function App() {
+  return <Progress thickness="large" label="Loading" />;
+}
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+- Indeterminate Progress
+  - The default Progress that animates indefinitely
+- Determinate Progress
+  - The determinate form of the Progress component that incrementally loads from 0% to 100%
+
+### Shape
+
+The Progress is represented as a rounded rectangular area with an inner animated bar that either travels across the area indefinitely or animates up till a specified point
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+### Slots
```

Comment:
We might want to use `Field` to add support for the label and description. It also looks like the visual spec includes support for error and success states, which the Field component already supports.

That would mean:
* The `Progress` control only has two slots:
  * `root` (which would be the track)
  * `bar` (child of root)
* There's a new `ProgresField` control that wraps `Progress` and adds the features of Field: label, helperText, validationState, etc.

That would follow the pattern of our other form controls, which (mostly) don't include labels built-in. And it would also benefit from sharing all of the features of Field.

Reply:
I agree with this but I think we discussed maybe making those changes to both spec and implementation in a separate PR.